Ubuntu 16.04 Deprecation (Azure Pipelines)

Microsoft Azure is letting us know they don't want us to use Ubuntu 16.04LTS anymore by breaking our CI:

https://dev.azure.com/ComputationalCryoEM/Aspire-Python/_build/results?buildId=3560&view=results

```
##[error]This is a scheduled Ubuntu 16.04 brownout. Ubuntu 16.04 LTS environment will be removed on October 18, 2021. For more details, see https://github.com/actions/virtual-environments/issues/3287
,##[error]The remote provider was unable to process the request.
```

We'll want to update our stable ubuntu to 18.04 LTS (Bionic Beaver) for Azure.  It looks like our other uses of Ubuntu is `lastest`, which is fine. See:

https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/master/azure-pipelines.yml
https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/master/.github/workflows/workflow.yml

@j-c-c since I gave the last CI issue to Chris, I'm thinking you can take this one.

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -31,9 +31,9 @@ jobs:
     displayName: pytest
 
 - job:
-  displayName: ubuntu-16.04
+  displayName: ubuntu-18.04
   pool:
-    vmImage: 'ubuntu-16.04'
+    vmImage: 'ubuntu-18.04'
   strategy:
     matrix:
       Python36:

